src/components/structures/RoomView.tsx

```diff
@@ -1087,7 +1087,7 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
             case "picture_snapshot":
                 ContentMessages.sharedInstance().sendContentListToRoom(
                     [payload.file],
-                    this.state.room.roomId,
+                    this.getRoomId(),
```

Comment:
this is still failing. Likewise various similar references below.

Reply:
This was intentional, consolidates everything to use this single method which in future will be made non-optional and to throw an error if no roomId so it is picked up by the error boundary

Reply:
This was deferred to a later task, this just consolidates the handling